lib/aggregate/attribute/date_time.rb

```diff
@@ -11,7 +11,7 @@ def load(value)
 
   def store(value)
     if @options[:format]
-      value.utc.to_s(@options[:format])
+      value.utc.to_formatted_s(@options[:format])
```

Comment:
In Rails 6, `to_formatted_s` is just an alias to `to_s`, however in Rails 7.1, this was changed to no longer override `to_s` and be a completely separate method.

https://apidock.com/rails/v7.0.0/ActiveSupport/TimeWithZone/to_s